jolt-core/src/jolt/vm/rv32i_vm.rs

```diff
@@ -189,8 +191,26 @@ mod tests {
         );
     }
 
+    // TODO(#343): Solve bug, reimplement.
```

Comment:
#343 should be a one-line fix,
Change [this](https://github.com/a16z/jolt/blob/main/jolt-core/src/jolt/vm/timestamp_range_check.rs#L719) to:
```
[1 + MEMORY_OPS_PER_INSTRUCTION + 5..1 + 2 * MEMORY_OPS_PER_INSTRUCTION + 5];
```

(and optionally add that assert to hyrax)